querybook/webapp/components/TemplateDataDocButton/TemplateDataDocButton.tsx

```diff
@@ -20,12 +22,56 @@ export const TemplateDataDocButton: React.FunctionComponent<IProps> = ({
 }) => {
     const [showTemplateForm, setShowTemplateForm] = React.useState(false);
 
+    const infoDOM = (
```

Comment:
can you extract this into a template info button component? will need it since we are going to have to display template config outside of a modal